aqt/installer.py

```diff
@@ -70,7 +70,40 @@ class Cli:
     __slot__ = ["parser", "combinations", "logger"]
 
     def __init__(self):
-        self._create_parser()
+        parser = argparse.ArgumentParser(
+            prog="aqt",
+            description="Another unoffical Qt Installer.\n"
+            "aqt help you install Qt SDK, tools, examples and others\n",
+            formatter_class=argparse.RawTextHelpFormatter,
+            add_help=True,
+        )
+        parser.add_argument(
+            "-c",
+            "--config",
+            type=argparse.FileType("r"),
+            help="Configuration ini file.",
+        )
+        subparsers = parser.add_subparsers(
+            title="subcommands",
+            description="aqt accept several subcommands\n"
+            "install-* subcommands are installer of components\n"
+            "list-* subcommands are show available compoenets\n\n"
+            "commands {install|tool|src|examples|doc} are oblesolete now\n",
```

Comment:
The word 'compoenets' should be 'components'.
The word 'oblesolete' should be spelled 'obsolete'.

I was expecting this message to say 'deprecated', as in, 'marked for removal at some point in the future'. This isn't always true, but 'deprecated' usually means 'a feature that exists for now, but will be removed in the future,' and 'obsolete' often means 'removed, but exists in the documentation for historical purposes'. I think it would be helpful to add some clarification as to what the message means by 'obsolete'.